portal/templates/utilisation-report-service/utilisation-report-upload/upload-previous-utilisation-reports.njk

```diff
@@ -0,0 +1,89 @@
+{% extends "index.njk" %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% set nextDueReportPeriod = [dueReports[0].month, " ", dueReports[0].year] | join %}
+{% set genericWarningText = "There are overdue reports, please send them as soon as possible." %}
+{% set warningText %}
+    {{ genericWarningText if dueReports.length > 2 else [nextDueReportPeriod, " report is overdue, please send it as soon as possible."] | join }}
+{% endset %}
+{% block pageTitle %}Utilisation Report Upload{% endblock %}
+{% block content %}
+    {% if errorSummary %}
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: errorSummary,
+          attributes: {
+            'data-cy': 'error-summary'
+          }
+        }) }}
+    {% endif %}
+    <h1 class="govuk-heading-l" data-cy="main-heading">
+        General Export Facility reporting
+    </h1>
+    {{ govukWarningText({
+        text: warningText,
+        iconFallbackText: "Warning.",
+        attributes: {
+        'data-cy': 'warning-text' }
+    }) }}
+    <ul class="govuk-list govuk-list--bullet" data-cy="due-reports-list">
+        {% for dueReport in dueReports.slice(0, -1) %}
+            <li data-cy="list-item-{{ dueReport.month }}-{{ dueReport.year }}__overdue">
+                {{ dueReport.month }} {{ dueReport.year }} report is overdue
+            </li>
+        {% endfor %}
+        <li data-cy="list-item-{{ dueReports.at(-1).month }}-{{ dueReports.at(-1).year }}__now-due">
+            {{ reportPeriod }} report is now due
+        </li>
+    </ul>
+    <div class="govuk-inset-text" data-cy="inset-text">
+        {% if dueReports.length > 2 %}
+            Once you've sent the {{ nextDueReportPeriod }} report, you can send subsequent reports.
+        {% else %}
+            Once you've sent the {{ nextDueReportPeriod }} report, you can send the {{ reportPeriod }} report.
+        {% endif %}
+    </div>
+    <p class="govuk-body">For each issued facility you must provide:</p>
+    <ul class="govuk-list govuk-list--bullet">
+        <li>facility utilisation accurate for the report month-end date</li>
+        <li>the total fees accrued</li>
+        <li>the fee payable to UKEF</li>
+    </ul>
+    <p class="govuk-body">If there has been no movement against a facility you must still report this.</p>
```

Comment:
All of this should be roughly the same as the other njk template. Could we look into either:
a) using a partial to commonise a lot of this template
b) using the same njk template with some conditional rendering for the top part where we have the different copy if we need to upload multiple reports
I think I'd prefer b) if possible and I hope it should be!